### DISCUSSION TITLE
HAMA or AMSA

### DISCUSSION THREAD

**Discussion by Snake3y3s** [ZeroQI/Hama.bundle#568] [Q&A]
So there is both the HAMA plex agent and the AMSA plex agent... as far as I understand, the AMSA agent uses the HAMA agent AND TVDB. 
So what is the point of having HAMA then?

Or can I install both or is it unnecessary? 
I,E. if AMSA uses more sources and includes HAMA, then why would I use HAMA?

**Comment by ZeroQI** [comment]
Never tried AMSA: https://github.com/Dingmatt/AMSA
- He started his project afterwars inspired by Hama and using my scanner, so without HAMA no AMSA
- uses anidb and tvdb, and hasn't be updated in two years... 

Please ask the AMSA maintainer, after all he copied me, not the other way around

**Reply by Snake3y3s** [reply]
Okay, Thank you
I will go ask in his discussion too :)
